src/api/modelTypes.js

```diff
@@ -211,28 +211,6 @@ export type UserGroup = $ReadOnly<{|
   name: string,
 |}>;
 
-/**
- * Specifies user status related properties
- * @prop away - present if we are to override user's presence status
- *   * This is the "user status" / "unavailable" feature added in early 2019.
- *     (At time of writing, there are no docs to link to.)
- * @prop status_text - a string representing information the user decided to
- *   manually set as their 'current status'
- */
-export type UserStatus = $ReadOnly<{|
```

Comment:
Dang, too bad this doesn't turn out to quite align between the initial data and the event.

Let's have "see also" comments pointing bidirectionally between those, though, to help the reader at least see what to compare to.

Reply:
Yeah, possibly. Are there other examples you have in mind, in addition to `Message`?

In the case of `Message`, the main difference within the API is on `flags`:
```js
  /**
   * The `flags` story is a bit complicated:
   *  * Absent in `event.message` for a `message` event... but instead
   *    (confusingly) there is an `event.flags`.
   *  * Present under `EVENT_NEW_MESSAGE`.
   *  * Present under `MESSAGE_FETCH_COMPLETE`, and in the server `/message`
   *    response that action is based on.
   *  * Absent in the Redux `state.messages`; we move the information to a
   *    separate subtree `state.flags`.
   */
  flags?: $ReadOnlyArray<string>,
```
where it feels to me like a mistake, not intentional. Though that doesn't mean we shouldn't distinguish the types; perhaps we should.

(In the case of `Message` I definitely wouldn't want to duplicate writing down all those other properties, though, because there are so many plus there's the subtleties in PMs vs. stream messages. We'd want to do something with spreads.)